r/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM rocker/r-ver:4.0.2
```

Comment:
I would suggest to use "official" alpine-based `RestRserve` image: `rexyai/restrserve:0.3.0-minimal` - https://hub.docker.com/r/rexyai/restrserve/tags

Reply:
I prefer to use this one. The idea is to use the same docker (when we will add other r based framework, such as https://github.com/rstudio/plumber)